# DP: backport of trunk r204854
# DP: fixes relocation errors when linking large libs (smokeqt, python-qt4)

--- a/src/config/picflag.m4
+++ b/src/config/picflag.m4
@@ -48,9 +48,6 @@ case "${$2}" in
     i[[34567]]86-*-* | x86_64-*-*)
 	$1=-fpic
 	;;
-    m68k-*-*)
-	$1=-fpic
-	;;
     # FIXME: Override -fPIC default in libgcc only? 
     sh-*-linux* | sh[[2346lbe]]*-*-linux*)
 	$1=-fpic
